modules

```diff
@@ -10,15 +10,9 @@ GLUON_SITE_FEEDS='community ssidchanger ffmuc'
 
 ##	PACKAGES_FFMUC_REPO
 #		the  git repository from where to clone the package feed
-PACKAGES_FFMUC_REPO=https://github.com/freifunkMUC/gluon-packages.git
-
-##	PACKAGES_FFMUC_COMMIT
-#		the version/commit of the git repository to clone
-PACKAGES_FFMUC_COMMIT=c1c668905b5dc9cc69acb663569a1905f003ffb7
-
-##  PACKAGES_FFMUC_BRANCH
-#   the branch to check out
-PACKAGES_FFMUC_BRANCH=main
+PACKAGES_FFMUC_REPO=https://github.com/grische/gluon-packages.git
+PACKAGES_FFMUC_COMMIT=22c79c3083f314c34fb7ba26b7c256fde1b47c75
```

Comment:
you need to bump this commit?

https://github.com/grische/gluon-packages/commit/1e52fc5eacb3c1092e619b082426472057d79260

Reply:
No, not until the package is reviewed and merged.